docs/CeedlingPacket.md

```diff
@@ -985,6 +985,38 @@ that you execute on target hardware).
 
   **Default**: FALSE
 
+  ---
+
+  **Note:**
+
+    The configuration can be combined together with
+    
+    ```
+    :test_runner:
+        :cmdline_args: true
+    ```
+
+    After setting **cmdline_args** to **true**, the debuger will will execute each of test
```

Comment:
```suggestion
    After setting **cmdline_args** to **true**, the debuger will execute each of test
```

Reply:
Hi @Letme,

I missed this during changes. Fixed.